test/test_open.py

```diff
@@ -227,4 +257,8 @@ def test_tempfile_perms(self, safer_open):
             perms.append(os.stat(filename).st_mode)
 
         assert perms == [perms[0]] * len(perms)
-        assert perms[0] in (0o100644, 0o100664)
+        # The octal value for read & write permissions across all groups and users in Windows is 0o100666
```

Comment:
Quibble: s/&/and/

(That's old-skool meaning "in this line, replace `&` with `and`.)